tests/gauge_alg_test.cpp

```diff
@@ -172,7 +173,7 @@ class GaugeAlgTest : public ::testing::Test {
   virtual void TearDown() {
     auto detu = getLinkDeterminant(*U);
     double2 tru = getLinkTrace(*U);
-    printfQuda("Det: %.16e:%.16e\n", detu.x, detu.y);
+    printfQuda("Det: %.16e:%.16e\n", double(detu.x), double(detu.y));
```

Comment:
use real/imag instead of this cast

Reply:
I think with detu the auto is a double2 and the `tru` is explicitly a double2. However I could rework it so that these initialize a `complex<double>` from which I can use real() and imag(). Fixed in fe2361790